product_docs/docs/pgd/5.6/commit-scopes/commit-scopes.mdx

```diff
@@ -175,4 +175,52 @@ SELECT bdr.alter_node_group_option(
 );
 ```
 
+## Degrade options
 
+CAMO, SYNCHRONOUS_COMMIT, and GROUP COMMIT each have rule degrade options.
+
+### CAMO 
+
+For [CAMO](pgd/latest/reference/commit-scopes/#camo), the options are simpler in that you can only degrade to asynchronous operation. 
```

Comment:
simpler than what? I'd move this to the end, after Synchronous Commit and Group Commit.

Reply:
Done. 👍